### PR TITLE
Jest tests

### LINKED ISSUE #7: Install Jest

### PR DESCRIPTION
**Pull request by ma-oliveiramarques**
Close #7

### CODE REVIEW

**Comment by ma-oliveiramarques** [comment]
Close #28

**Comment by pncsoares** [comment]
@ma-oliveiramarques where are the `*.test.js` file?

File: src/func.test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const allOperations = require('./func');
```
Comment (pncsoares) [inline src/func.test.js:1]: Did you tested the tests?
What was the result?

**Review by pncsoares — COMMENTED** [review]
Why do you need the `main.js` file?

**Comment by ma-oliveiramarques** [comment]
Close #8

**Comment by ma-oliveiramarques** [comment]
Close #9

**Review by pncsoares — APPROVED** [review]
Well done! 👏

**Comment by ma-oliveiramarques** [comment]
Close #11

**Review by pncsoares — DISMISSED** [review]
Well done! 👏

**Comment by ma-oliveiramarques** [comment]
Close #10

**Review by pncsoares — APPROVED** [review]

**Pull request merged**